stencil/src/components/pulumi-api-doc-nav-tree/pulumi-api-doc-nav-tree.tsx

```diff
@@ -33,8 +35,14 @@ export class PulumiApiDocNavTree {
     // even if it has not been clicked by the user.
     shouldNodeBeSelected(nodeHref) {
         // To compare the current path to the href, we strip the starting and ending slashes.
-        const formattedCurrentPath = window.location.pathname.split("/").filter(item => item !== "").join("/");
-        const formattedNodeHref = nodeHref.split("/").filter(item => item !== "").join("/");
+        const formattedCurrentPath = window.location.pathname
```

Comment:
This isn't changed, I just ran Prettier on the file, which I must have forgotten last time.